modules/index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+
```

Comment:
- [ ] Please refrain from using `eslint-disable` on the whole file, as it defies the purpose of using linters in the first place.